ci/testkomodo.sh

```diff
@@ -25,6 +25,7 @@ start_tests () {
     # Allow xvfb to find a new server
     pushd ${CI_TEST_ROOT}/tests
     xvfb-run -s "-screen 0 640x480x24" --auto-servernum python -m \
-    pytest -m "not requires_window_manager"
+        pytest -m "not requires_window_manager"
+    pytest -m "requires_eclipse" --eclipse-simulator
```

Comment:
The  part `-m "requires_eclipse"` does the subset selection of 17 eclipse related tests.  These 17 are skipped by default, but the option `--eclipse-simulator` turns them on.

Reply:
Right, now it makes sense :+1: